packages/map-template/src/components/ContactActionButton/ContactActionButton.scss

```diff
@@ -0,0 +1,41 @@
+.contact-action-button {
```

Comment:
We have a collection of custom CSS properties that we should use whenever applicable. 

For inspiration, look at for example the SCSS file for the QR Code dialog component: https://github.com/MapsPeople/web-ui/blob/main/packages/map-template/src/components/QRCodeDialog/QRCodeDialog.scss

So for example instead of `padding: 8px;` it should be `padding: var(--spacing-x-small);`